var/spack/repos/builtin/packages/py-neurotools/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNeurotools(PythonPackage):
+    """A collection of tools for representing and anlyzing neuroscientific
+    data."""
+
+    homepage = "http://neuralensemble.org/NeuroTools"
+    pypi = "neurotools/NeuroTools-0.3.1.tar.gz"
+
+    version("0.3.1", sha256="a459420fc0e9ff6b59af28716ddb0c75d11a63b8db80a5f4844e0d7a90c2c653")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-scipy", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-matplotlib", type=("build", "run"))
+    depends_on("py-tables", type=("build", "run"))
+    depends_on("py-pyaml", type=("build", "run"))
+
+    patch("neurotools-0.3.1.patch", when="@0.3.1")
```

Comment:
Still need a link to where this patch comes from or some discussion of why it is needed

Reply:
Comes originally from https://github.com/BlueBrain/bbp-nixpkgs/pull/1026 (sorry, absolutely useless PR description)

The original code in essence tries to do this:
```
>>> np.random.RandomState().exponential(1.5, np.ceil(3.6))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mtrand.pyx", line 540, in numpy.random.mtrand.RandomState.exponential
  File "_common.pyx", line 630, in numpy.random._common.cont
TypeError: expected a sequence of integers or a single integer, got '4.0'
```
so patching in `int()` will avoid this error (the parameter in question are the size of the returned `numpy.array` containing random numbers).

Reply:
Thanks, seems like the library is no longer under development so there's no need to report this to the developers